Assets/PatchKit Patcher/Scripts/AppData/Local/TemporaryDirectory.cs

```diff
@@ -13,7 +14,9 @@ public sealed class TemporaryDirectory : IDisposable
     {
         public string Path { get; private set; }
 
-        public TemporaryDirectory([NotNull] string path)
+        private bool _keep = false;
+
+        private TemporaryDirectory([NotNull] string path)
```

Comment:
Because of this change I think we should add behavior of clearing previous directory content in constructor, since temp directories might not get removed.

@genail @Dzejkop what do you think?

Reply:
@witcher112 it's a good idea actually